File: data/EXP01/golang_no_system_var/2232/2232.output_1.go
func minimizeResult(expression string) string {
    var result string
    n := len(expression)
    for i := 0; i < n; i++ {
        if expression[i] == '+' {
            result = expression[:i-1] + "(" + expression[i-1:i+2] + ")" + expression[i+2:]
            break
        }
    }
    return result
}

File: data/EXP01/python/2232/2232.output_1.py
# @lc app=leetcode id=2232 lang=python3
class Solution:
    def minimizeResult(self, expression: str) -> str:
        n = len(expression)
        smallest_val = eval(expression)  # evaluate the given expression without parentheses
        smallest_exp = expression

        for i in range(1, n-1):
            num1 = expression[:i]
            num2 = expression[i+1:]
            new_exp = num1 + '(' + num2 + ')'  # add parentheses at position i
            val = eval(new_exp)  # evaluate the new expression

            if val < smallest_val:
                smallest_val = val
                smallest_exp = new_exp

        return smallest_exp